composer.json

```diff
@@ -17,7 +17,9 @@
     "require": {
         "php": ">=5.3.3",
         "symfony/symfony": "~2.6",
-        "doctrine/orm": "~2.2,>=2.2.3",
+        "symfony/symfony": "2.6.*",
+        "doctrine/orm": "~2.2,>=2.2.3,<2.5",
+        "doctrine/dbal": "<2.5",
```

Comment:
Because it has a severe BC break, breaking all installations. See https://github.com/symfony/symfony-standard/pull/774 for more information.

They (@ocramius) are working hard to fix it, after which we can allow 2.5 again.